test/vcf/record_test.cpp

```diff
@@ -481,7 +481,7 @@ namespace ebi
                 vcf::INFO,
                 {
                     { vcf::ID, vcf::AF },
-                    { vcf::NUMBER, vcf::A },
+                    { vcf::NUMBER, "A" },
```

Comment:
oops, this usage was correct as vcf::A, I thought there was only one usage of this at the beginning of this file